_dprhtml/js/dpr_pal.js

```diff
@@ -382,3 +382,7 @@
 // NOTE: Keep this out side the above.
 window.addEventListener('focus', () => { DPR_PAL.windowHasFocus = true; });
 window.addEventListener('blur', () => { DPR_PAL.windowHasFocus = false; });
+
+if (typeof module !== "undefined") {
```

Comment:
Manual exports like this make me realize that I have definitely taken modern JS modules and import functionality for granted 😅.